### PR TITLE
Wait for screen change after selecting addon

### PR DESCRIPTION
**Pull request by dzedro**
Avoid [fail](https://openqa.suse.de/tests/890007)

### CODE REVIEW

File: lib/registration.pm [hunk 1]
```diff
@@ -231,15 +231,15 @@ sub select_addons_in_textmode {
     if ($flag) {
         send_key_until_needlematch 'scc-module-area-selected', 'tab';
         send_key_until_needlematch "scc-module-$addon",        'down';
-        send_key 'spc';
+        wait_screen_change { send_key 'spc' };
```
Comment (okurz) [inline lib/registration.pm]: But we are doing the "wait_still_screen" just afterwards. That does not make sense to me

Comment (mimi1vx) [inline lib/registration.pm]: @okurz https://openqa.suse.de/tests/890007#step/scc_registration/7

Comment (okurz) [inline lib/registration.pm]: yes, I have seen this link but still "wait_screen_change+wait_still_screen". why should it make sense?

Comment (mitiao) [inline lib/registration.pm]: I think the issue is caused by wrong needle:
https://openqa.suse.de/tests/890007#step/scc_registration/6
The test actually didn't select LTSS addon

Comment (dzedro) [inline lib/registration.pm]: wait_still_screen 1; is like joke, I will remove it

Comment (dzedro) [inline lib/registration.pm]: lol yes the needle is wrong ... well this would make it more robust but leave it for now ><

**Comment by dzedro** [comment]
I deleted that nonsense needle

**Pull request closed without merging**